packages/core/src/util/schema.ts

```diff
@@ -27,7 +27,7 @@ import find from 'lodash/find';
 import { JsonSchema } from '../models';
 
 export const getFirstPrimitiveProp = (schema: any) => {
-  if (schema.properties) {
+  if (schema && schema.properties) {
```

Comment:
```suggestion
  if (schema?.properties) {
```

Couldn't this be done like this?